sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClient.java

```diff
@@ -423,75 +411,74 @@ public KeyVaultSecret restoreSecretBackup(byte[] backup) {
+     * value of its latest version.</p>
+     *
      * {@codesnippet com.azure.security.keyvault.secretclient.listSecrets}
      *
-     * <p><strong>Code Samples to iterate over secrets by page</strong></p>
-     * <p>It is possible to get full secrets with values from this information. Iterate over all the {@link SecretProperties
-     * secret} by page and call {@link SecretClient#getSecret(String, String)}. This will return the
-     * {@link KeyVaultSecret secret} with value included of its latest version.</p>
+     * <p><strong>Iterate over secrets by page</strong></p>
+     * <p>The snippet below loops over each {@link SecretProperties secret} by page and calls
+     * {@link #getSecret(String, String) getSecret(String, String)}. This gets the {@link KeyVaultSecret secret} and the
+     * value of its latest version.</p>
+     *
      * {@codesnippet com.azure.security.keyvault.secretclient.listSecrets.iterableByPage}
      *
-     * @return {@link PagedIterable} of {@link SecretProperties} of all the secrets in the vault. The {@link SecretProperties}
-     *     contains all the information about the secret, except its value.
+     * @return {@link PagedIterable} of {@link SecretProperties} of all the secrets in the vault. The
+     *     {@link SecretProperties} contains all the information about the secret, except its value.
      */
     public PagedIterable<SecretProperties> listPropertiesOfSecrets() {
         return listPropertiesOfSecrets(Context.NONE);
     }
 
     /**
-     * List the secrets in the key vault. The list Secrets operation is applicable to the entire vault. The individual
-     * secret response in the list is represented by {@link SecretProperties} as only the secret identifier and its
-     * attributes are provided in the response. The secret values and individual secret versions are not listed in the
-     * response. This operation requires the {@code secrets/list} permission.
-     *
-     * <p><strong>Code Samples to iterate over secrets by page</strong></p>
-     * <p>It is possible to get full secrets with values from this information. Loop over the {@link SecretProperties secret}
-     * and call {@link SecretClient#getSecret(String, String)}. This will return the {@link KeyVaultSecret secret} with
-     * value included of its latest version.</p>
+     * Lists secrets in the key vault. Each {@link SecretProperties secret} returned only has its identifier and
+     * attributes populated. The secret values and their versions are not listed in the response.
+     * This operation requires the {@code secrets/list} permission.
+     *
+     * <p><strong>Iterate over secrets and fetch their latest value</strong></p>
+     * <p>The snippet below loops over each {@link SecretProperties secret} and calls
+     * {@link #getSecret(String, String) getSecret(String, String)}. This gets the {@link KeyVaultSecret secret} and the
```

Comment:
Other places, link is casted to getSecret().
Did you intend to cast to getSecret(String, String) in other places ?